google-api-client/src/main/java/com/google/api/client/googleapis/GoogleUtils.java

```diff
@@ -29,42 +30,50 @@
  */
 public final class GoogleUtils {
 
+  // NOTE: toString() so compiler thinks it isn't a constant, so it won't inline it
+  // {x-version-update-start:google-api-client:current}
+  /** Current release version. */
+  public static final String VERSION = "1.30.3-SNAPSHOT".toString();
+  // {x-version-update-end:google-api-client:current}
+
   // NOTE: Integer instead of int so compiler thinks it isn't a constant, so it won't inline it
   /**
    * Major part of the current release version.
    *
    * @since 1.14
    */
-  public static final Integer MAJOR_VERSION = 1;
+  public static final Integer MAJOR_VERSION;
 
   /**
    * Minor part of the current release version.
    *
    * @since 1.14
    */
-  public static final Integer MINOR_VERSION = 26;
+  public static final Integer MINOR_VERSION;
 
   /**
    * Bug fix part of the current release version.
    *
    * @since 1.14
    */
-  public static final Integer BUGFIX_VERSION = 0;
+  public static final Integer BUGFIX_VERSION;
 
-  /** Current release version. */
-  // NOTE: toString() so compiler thinks it isn't a constant, so it won't inline it
-  public static final String VERSION = (MAJOR_VERSION + "." + MINOR_VERSION + "." + BUGFIX_VERSION
-      + "-SNAPSHOT").toString();
+  static {
+    Matcher versionMatcher =
```

Comment:
Would just add a small test for the matcher.

Reply:
Done. Moved the pattern into a package private constant visible for testing.